src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/BrowseBuildingFloors/BrowseBuildingFloors.xaml.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Mapping.Floor;
+using Esri.ArcGISRuntime.Portal;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace ArcGISRuntime.WPF.Samples.BrowseBuildingFloors
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        name: "Browse building floors",
+        category: "Map",
+        description: "Display and browse through building floors from a floor-aware web map.",
+        instructions: "Use the spinner to browse different floor levels in the facility. Only the selected floor will be displayed.",
+        tags: new[] { "building", "facility", "floor", "floor-aware", "floors", "ground floor", "indoor", "level", "site", "story" })]
+    public partial class BrowseBuildingFloors
+    {
+        private const string _portalItem = "f133a698536f44c8884ad81f80b6cfc7";
+        private FloorManager _floorManager;
+
+        // Collection of floors.
+        private readonly Dictionary<string, FloorLevel> _floorOptions = new Dictionary<string, FloorLevel>();
+
+        public BrowseBuildingFloors()
+        {
+            InitializeComponent();
+            _ = Initialize();
+        }
+
+        private async Task Initialize()
+        {
+            try
+            {
+                ArcGISPortal portal = await ArcGISPortal.CreateAsync();
+
+                // Get the portal item for a web map using its unique item id.
+                PortalItem mapItem = await PortalItem.CreateAsync(portal, _portalItem);
+                Map map = new Map(mapItem);
+
+                MyMapView.Map = map;
+
+                // Map needs to be loaded in order for floormanager to be used.
+                await MyMapView.Map.LoadAsync();
+                await MyMapView.Map.FloorManager.LoadAsync();
```

Comment:
FloorManager happens to not be null for this web map, but it could be null for others.

I'd prefer an explicit check to make sure the floor manager isn't null, rather than implicitly relying on an exception being thrown.

Reply:
@nCastle1 I have the code also checking if the the FloorManager fails to load. Should I add an extra term to check if the FloorManger is null as well?

Reply:
Yes, this line will throw if you try with a map that isn't floor-aware because FloorManager will still be null